blog/2022/05-10-resoto-graph-traversal/index.md

```diff
@@ -0,0 +1,119 @@
+---
+authors: [matthias]
+tags: [graph, search, digitalocean]
+---
+
+# A walk in the graph
+
+Resoto uses a [directed graph](https://en.wikipedia.org/wiki/Directed_graph) to represent your infrastructure resources as nodes and relationships between them as edges. It stores that resource data in a graph database, allowing you to search the graph efficiently. Why should I care, you ask? To answer this question, I would like to present an example that demonstrates the power of this approach.
```

Comment:
this paragraph would benefit from splitting it up.

... search graph efficiently.

Why should I care, you ask?

In this post, I'm sharing an example that demonstrates the power of this approach.